tests/BenchmarkDotNet.Tests/Validators/ConfigCompatibilityValidatorTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Order;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Validators;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace BenchmarkDotNet.Tests.Validators
+{
+    public class ConfigCompatibilityValidatorTests
+    {
+        [Fact]
+        public void JoinedBenchmarksShouldntHaveDifferentOrderers()
```

Comment:
Could you please add one more test for a case where there are no extra orderders defined and the validator does not return any errors?